lib/public/views/Runs/RunsPerSimulationPass/RunsPerSimulationPassOverviewPage.js

```diff
@@ -84,7 +89,18 @@ export const RunsPerSimulationPassOverviewPage = ({
             exportRunsTriggerAndModal(runsPerSimulationPassOverviewModel, modalModel),
         ]),
         h('.flex-column.w-100', [
-            table(remoteRuns, activeColumns, null, { profile: 'runsPerSimulationPass' }),
+            table(
+                remoteDetectors.match({
+                    Success: () => remoteRuns,
+                    Other: () => remoteDetectors,
+                }),
```

Comment:
Same here

Reply:
because previous was wrong, anyway it's revoked